decidim-core/spec/services/decidim/open_data_exporter_spec.rb

```diff
@@ -146,7 +146,7 @@
         end
 
         it "includes the meetings data" do
-          expect(csv_data).to include(meeting.title["en"])
+          expect(csv_data).to include(translated(meeting.title).gsub(/"/, '""'))
```

Comment:
This is how its done in develop
```suggestion
          expect(csv_data).to include(meeting.title["en"].gsub(/"/, '""'))
```